src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Mappers/Apprentice/RecognisePriorLearningViewModelToResultMapper.cs

```diff
@@ -1,35 +1,47 @@
 ﻿using SFA.DAS.CommitmentsV2.Api.Client;
 using SFA.DAS.CommitmentsV2.Shared.Interfaces;
+using SFA.DAS.ProviderCommitments.Features;
+using SFA.DAS.ProviderCommitments.Infrastructure.OuterApi;
+using SFA.DAS.ProviderCommitments.Infrastructure.OuterApi.Requests.DraftApprenticeship;
+using SFA.DAS.ProviderCommitments.Infrastructure.OuterApi.Requests.DraftApprenticeships;
+using SFA.DAS.ProviderCommitments.Interfaces;
 using SFA.DAS.ProviderCommitments.Web.Models;
 using System.Threading;
 using System.Threading.Tasks;
-using SFA.DAS.Authorization.Services;
-using SFA.DAS.ProviderCommitments.Features;
 
 namespace SFA.DAS.ProviderCommitments.Web.Mappers.Apprentice
 {
-    public class RecognisePriorLearningViewModelToResultMapper : IMapper<RecognisePriorLearningViewModel, RecognisePriorLearningResult>
+    public class PriorLearningDetailsViewModelToResultMapper : IMapper<PriorLearningDetailsViewModel, RecognisePriorLearningResult>
     {
         private readonly ICommitmentsApiClient _commitmentsApiClient;
-
-        public RecognisePriorLearningViewModelToResultMapper(ICommitmentsApiClient commitmentsApiClient)
-            => _commitmentsApiClient = commitmentsApiClient;
-
-        public async Task<RecognisePriorLearningResult> Map(RecognisePriorLearningViewModel source)
+        private readonly IOuterApiClient _outerApiClient;
+        private readonly DAS.Authorization.Services.IAuthorizationService _authorizationService;
+        public PriorLearningDetailsViewModelToResultMapper(ICommitmentsApiClient commitmentsApiClient, DAS.Authorization.Services.IAuthorizationService authorizationService, IOuterApiClient outerApiClient)
         {
-            var update = _commitmentsApiClient.RecognisePriorLearning(
+            _commitmentsApiClient = commitmentsApiClient;
+            _authorizationService = authorizationService;
+            _outerApiClient = outerApiClient;
+        }
+        public async Task<RecognisePriorLearningResult> Map(PriorLearningDetailsViewModel source)
+        {
+            var update = _commitmentsApiClient.PriorLearningDetails(
```

Comment:
Could this request not go via the outer api? I can see that we used to call the inner api directly here, but it seems like payload has dramatically changed, in which case migration to the new pattern is preferable.

Reply:
Yes, however, it was pre-existing code and Ali was trying to tidy up code, removed it and then re-added (because it weas needed when the toggle is off)>

This code can be fully removed when RPL V2 is released and turned on. I will add a story to our backlog